probe-rs/src/architecture/riscv/dtm/jtag_dtm.rs

```diff
@@ -90,8 +92,8 @@ impl<'probe> JtagDtm<'probe> {
         Self { probe, state }
     }
 
-    fn transform_dmi_result(response_bytes: Vec<u8>) -> Result<u32, DmiOperationStatus> {
-        let response_value = u128_from_data(&response_bytes);
+    fn transform_dmi_result(response_bits: &BitSlice) -> Result<u32, DmiOperationStatus> {
+        let response_value = response_bits.load_le::<u128>();
```

Comment:
This fails on wlink with ch32v003:  "cannot load 128 bits from a 160-bit region"
A lot of other instances in this file have non matching bit sizes.

Reply:
@Yatekii do you have one of these?

Reply:
I do not. I can order one so it's there when I get home.